src/libmongoc/tests/test-mongoc-uri.c

```diff
@@ -255,6 +255,15 @@ test_mongoc_uri_new (void)
    ASSERT (!mongoc_uri_new (
       "mongodb://localhost/?" MONGOC_URI_HEARTBEATFREQUENCYMS "=499"));
 
+   /* timeoutMS */
```

Comment:
Nit, is this comment needed?

Reply:
Removed

Reply:
Why not just create a new function? This one is already really long as is ([code smell: long method](https://refactoring.guru/smells/long-method)). From the link

> As a rule of thumb, if you feel the need to comment on something inside a method, you should take this code and put it in a new method.

I think that's applicable here. In general, this function is pretty hard to read given that you can't see the entire function at once (unless your monitor is ridiculously tall). I wouldn't be opposed to breaking this function up either.